internal/testutil/acc/independent_shard_scaling.go

```diff
@@ -0,0 +1,48 @@
+package acc
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/mongodb-forks/digest"
+)
+
+func GetIndependentShardScalingMode(ctx context.Context, projectID, clusterName string) (*string, *http.Response, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "https://cloud-dev.mongodb.com/test/utils/auth/groups/"+projectID+"/clusters/"+clusterName+"/independentShardScalingMode", http.NoBody)
```

Comment:
why are we doing this? isn't it in the SDK? is it in Preview?

Reply:
This is an endpoint only available in cloud-dev and specific for these tests, so it's not available in the SDK and it won't be in the future. That's the reason why we call it directly

Reply:
This means that the tests will fail in QA? @oarbusi

Reply:
Great point, didn't mention that the endpoint is available in QA also (basically the endpoint is not available in prod and is outside of Admin API so not available in the SDK). I did the necessary change so that all the tests are run in QA successfully in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2851/commits/306353830356b56ca9b4f5954bb9b8fd404f088c